### PR TITLE
Astro 1112 create login content

### PR DESCRIPTION
**Pull request by Design-Kat**
Added Sign in text content. 
Added License security add from Michelle.

### CODE REVIEW

File: _content/components/sign-in.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+Don't: Display sensitive data unmasked by default
+:::
+
+![Do: Use clear button labels that describe specific tasks like "Sign in" or "Update Password"](/img/components/component-template-do-1.png "Do: Something")
+Do: Use clear button labels that describe specific tasks like "Sign in" or "Update Password"
+
+![Don't: Use vague button labels like "Submit" to guide users through authentication flows](/img/components/component-template-dont-1.png "Don't: Something")
+Don't: Use vague button labels like "Submit" to guide users through authentication flows
+:::
+![Do: Use clear and concise messaging to help users understand what went wrong and give users steps to resolve the error.   e.g. let users  know when the caps lock is on.](/img/components/component-template-do-1.png "Do: Something")
```
Comment (amalyah) [inline _content/components/sign-in.md:109]: error. e.g. let users know when the caps lock is on. > error, e.g. let users know when the caps lock is on.

File: _content/components/sign-in.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+Do: Use clear and concise messaging to help users understand what went wrong and give users steps to resolve the error.   e.g. let users  know when the caps lock is on.
```
Comment (amalyah) [inline _content/components/sign-in.md:110]: Same change as listed above.

File: _content/components/sign-in.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+When a user wants to gain access to an application.
+When a user has logged out voluntarily. They will see a confirmation message after which they will be automatically redirected back to the login page.
+When a user has been logged out due to inactivity. In this scenario, when the user logs back in they should be redirected to the last page they were on, before being logged out.
+
+## Rules of Thumb
+
+-  Use “Sign in”, not Login, Log in or Log-in. (Note: When leaving an application, use “Sign off”, not Log off)
+-  Use a direct, actionable button name, such as Sign In, rather than a more generic “Submit” to post the form.
+-  Consult with your IT/Security team to ensure you are meeting your specific authentication best practices and security needs.
+-  Provide a way for users to sign-out and re-authenticate.
```
[the remark this answers is not in the document]
Comment (Design-Kat) [reply _content/components/sign-in.md:31]: Should have been Sign out, not sign off.

[the remark this answers is not in the document]
Comment (Design-Kat) [reply _content/components/sign-in.md:31]: > @Design-Kat Some review feedback for you on Login documentation.

@amalyah PR #157 created for these updates, please review when you can, thank you!

**Pull request merged**